lib/phoenix/controller.ex

```diff
@@ -133,6 +133,12 @@ defmodule Phoenix.Controller do
   @spec controller_template(Plug.Conn.t) :: binary
   def controller_template(conn), do: conn.private[:phoenix_template]
 
+  @compile {:inline, get_json_encoder: 0}
```

Comment:
I thought that since `get_json_encoder` is both small and (potentially) called frequently it would make a good candidate for inlining.